src/Card/Card.Image.js

```diff
@@ -0,0 +1,50 @@
+import { dcnb } from "cnbuilder";
+import React from "react";
+import PropTypes from "prop-types";
+import { imageAspectRatios } from "./Card.levers";
+
+/**
+ * Image component.
+ * @param {*} param0
+ * @returns
+ */
+export const Image = ({ image, aspectRatio, className }) => {
+  // Basic card image has aspect ratio 2x1
+  const imageAspectRatio = imageAspectRatios[aspectRatio];
+
+  // Remove alt.
+  if (image.alt) {
+    // eslint-disable-next-line no-param-reassign
+    delete image.alt;
+  }
+
+  return (
+    <div className={dcnb(imageAspectRatio, className)} aria-hidden="true">
+      <img {...image} alt="" className="su-object-cover" />
```

Comment:
If you're going to add a classname to this element you will have to remove `className` from the `image` prop to avoid duplication like we did with `alt`.

Reply:
Done